src/main/java/com/synopsys/integration/alert/web/controller/AuthenticationController.java

```diff
@@ -91,4 +99,22 @@ public LoginController(final LoginActions loginActions, final ResponseFactory re
             return responseFactory.createMessageResponse(HttpStatus.INTERNAL_SERVER_ERROR, ex.getMessage());
         }
     }
+
+    @PostMapping(value = "/resetPassword")
+    public ResponseEntity<String> resetPassword() {
+        return responseFactory.createBadRequestResponse(null, "Password Reset Error: A username must be specified");
+    }
+
+    @PostMapping(value = "/resetPassword/{username}")
+    public ResponseEntity<String> resetPassword(@PathVariable final String username) {
```

Comment:
Why have 2 endpoints defined here if the one without the username path variable just return a bad request? That could be the very first check in the correct method

Reply:
If we don't define the route without the path variable, it returns a 404